### ISSUE TITLE
Failure of http extension using installer

### ISSUE THREAD

**Issue by aum-tcm** [mlocati/docker-php-extension-installer#1032]
### Version of install-php-extensions

v.2.7.7

### Error description

Hi, first of all, thank you for maintaining this great tool.
I’ve been successfully using it to install the http extension for a while, but I recently encountered an issue where the installation now fails. I wanted to bring this to your attention.

An issue occurs when attempting to install the http extension via the mlocati/docker-php-extension-installer tool. The installation fails during the build process of a dependency, resulting in compilation errors related to the iconv function. Below are the error messages and details.

```
localconverter.c: In function 'localconverter_iconv_conv':
localconverter.c:602:21: error: passing argument 2 of 'iconv' from incompatible pointer type [-Wincompatible-pointer-types]
  602 |         iconv(ictx, (const char **)NULL, &inleft, &outbuf, &outleft);
      |                     ^
      |                     |
      |                     const char **
In file included from localconverter.c:195:
/usr/include/iconv.h:17:23: note: expected 'char ** restrict' but argument is of type 'const char **'
   17 | size_t iconv(iconv_t, char **__restrict, size_t *__restrict, char **__restrict, size_t *__restrict);
      |                       ^

localconverter.c:607:26: error: passing argument 2 of 'iconv' from incompatible pointer type [-Wincompatible-pointer-types]
  607 |         sz = iconv(ictx, (const char **)&inbuf, &inleft, &to, &outleft);
      |                          ^~~~~~~~~~~~~~~~~~~~~
      |                          |
      |                          const char **
/usr/include/iconv.h:17:23: note: expected 'char ** restrict' but argument is of type 'const char **'
   17 | size_t iconv(iconv_t, char **__restrict, size_t *__restrict, char **__restrict, size_t *__restrict);
      |                       ^

localconverter.c:633:26: error: passing argument 2 of 'iconv' from incompatible pointer type [-Wincompatible-pointer-types]
  633 |         sz = iconv(ictx, (const char **)NULL, &inleft, &to, &outleft);
      |                          ^
      |                          |
      |                          const char **
/usr/include/iconv.h:17:23: note: expected 'char ** restrict' but argument is of type 'const char **'
   17 | size_t iconv(iconv_t, char **__restrict, size_t *__restrict, char **__restrict, size_t *__restrict);
      |                       ^

make[1]: *** [Makefile:800: libidnkit_la-localconverter.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/tmp/src/tmp.oEpmPG/idnkit-2.3/lib'
make: *** [Makefile:329: install-recursive] Error 1
error building image: error building stage: failed to execute command: waiting for process to exit: exit status 2

```

I've tested different versions of the tool (v2.7.6 and v2.7.5) and PHP (8.4) without success

Thank you in advance for your time and support

### Docker image

php:8.3-fpm-alpine

### Minimal Dockerfile

```Dockerfile
FROM php:8.3-fpm-alpine

# Basics
RUN apk --update --no-cache add bash nano

# Add easy extension installer
ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
RUN chmod +x /usr/local/bin/install-php-extensions

# Install default required extensions
RUN install-php-extensions http
```

**Comment by mlocati** [comment]
Thanks for the report: I've just published version 2.7.8 which should fix this issue.

**Comment by aum-tcm** [comment]
Wow, indeed it works now. Thanks for support !
